apps/web/github.talon

```diff
@@ -2,7 +2,6 @@
 tag: browser
 browser.host: github.com
 -
-
```

Comment:
@wenkokke in grooming session we aligned on actually keeping one empty line after the `-` line, and no empty lines before.  Assuming you have no objections, I'm guessing that's an easy fix?  Sorry to go back and forth on this one 😅

Reply:
Easy change, can do. Probably best to check back if the other thing (mandatory “-“) is kinda important?